packages/cli/src/commands/export.js

```diff
@@ -0,0 +1,46 @@
+import { TARGETS } from '@nuxt/utils'
+import { common, locking } from '../options'
+import { createLock } from '../utils'
+
+export default {
+  name: 'export',
+  description: 'Export a static generated web application',
```

Comment:
I believe new command is to avoid breaking changes and something better for name. Being fullStatic or not can be an option for nuxt3

Reply:
Indeed, `nuxt export` is planned to be the new command to export (and will activate by default the full static option), `nuxt generate` will be deprecated since it also run the build which `nuxt export` does not.